ui/app/mainui/AppMain.qml

```diff
@@ -801,6 +802,21 @@ Item {
 
         // TODO: Remove this and adapt new mechanism to launch BuyModal as done for SendModal
         property BuyCryptoParamsForm buyFormData: BuyCryptoParamsForm {}
+
+        property SwapInputParamsForm swapFormData: SwapInputParamsForm {}
+
+        function launchSwap() {
+            swapFormData.selectedAccountAddress =
+                    SQUtils.ModelUtils.get(WalletStores.RootStore.nonWatchAccounts, 0, "address")
+            swapFormData.selectedNetworkChainId =
+                    SQUtils.ModelUtils.getByKey(WalletStores.RootStore.filteredFlatModel, "layer", 1, "chainId")
+            swapFormData.defaultFromTokenKey = Constants.swap.usdtTokenKey
```

Comment:
Can we default the tokens in the swapFormData component? I guess we'll need these defaults no matter where the Swap is started from.